pcdet/models/detectors/pv_rcnn_ssl.py

```diff
@@ -320,18 +320,18 @@ def forward(self, batch_dict):
     # TODO(farzad) Warning: this method has two types of stats: one maintaining their stats over batches, e.g., TP, FP
     #  and two reset state after each batch. Use update_metrics flag if you want call the method twice in different
     #  places during a batch.
-    def calc_statistics(self, pred_boxes, gt_boxes, update_metrics=True, tag=None, **kwargs):
-        assert len(pred_boxes) == len(gt_boxes)
-
+    def calc_statistics(self, pred_boxes_org, gt_boxes_org, update_metrics=True, tag=None, **kwargs):
+        assert len(pred_boxes_org) == len(gt_boxes_org)
+        
         statistics = defaultdict(list)
-        for i in range(len(pred_boxes)):
-            valid_preds_mask = pred_boxes[i].sum(dim=-1) > 0
-            valid_gts_mask = gt_boxes[i].sum(dim=-1) > 0
+        for i in range(len(pred_boxes_org)):
+            valid_preds_mask = pred_boxes_org[i].sum(dim=-1) > 0
+            valid_gts_mask = gt_boxes_org[i].sum(dim=-1) > 0
             num_gts = valid_gts_mask.sum()
             num_preds = valid_preds_mask.sum()
             if num_gts > 0 and num_preds > 0:
-                gt_boxes = gt_boxes[i, valid_gts_mask]
```

Comment:
Can you please rename `gt_boxes` and `pred_boxes` to `valid_gt_boxes` and `valid_pred_boxes` instead of method arguments?